### PR TITLE
Connection settings update for null check

### PR DESCRIPTION
**Pull request by ishachopra0401**
## Description of the change

Updated screenshots for connection settings and added 'Skip Verify' option

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: destinations/advertising/fb-custom-audience.mdx [hunk 1]
```diff
@@ -101,6 +104,11 @@ Even when disabled, Facebook does not accept any other schema fields coupled wit
 
 - **Disable Formatting**: Facebook has fixed data formats for all the allowed schema fields. If this option is enabled, RudderStack will not format the user data before sending it to Custom Audience.
 
+- **Skip Verify**: This option lets Rudderstack handle all the null/undefined user properties before uploading them to Facebook Custom Audience.
+    - When disabled, there is no impact.
```
Comment (ItsSudip) [inline destinations/advertising/fb-custom-audience.mdx:108]: Change this to something like this "When disabled, we will discard the entire batch of events if any of the user attributes are null according to the schema fields.". CC @utsabc

**Review by shrouti1507 — APPROVED** [review]
LGTM

File: destinations/advertising/fb-custom-audience.mdx (hunk 1 above)
Comment (shrouti1507) [reply destinations/advertising/fb-custom-audience.mdx:108]: The thing is we ourselves will not discard the whole batch. We will send without the field values that are entered as null. But the from destination end, the entire batch will be discarded. @ItsSudip Reason is, Facebook expects data for every schema field that has been chosen for the particular event.

Comment (utsabc) [reply destinations/advertising/fb-custom-audience.mdx:108]: @shrouti1507 We will straightaway reject if null values are present and skip verify is disabled. Events will not flow through transformer

Comment (utsabc) [reply destinations/advertising/fb-custom-audience.mdx:108]: P.S I agree with @ItsSudip the statement looks more simple and easy to understand

Comment (utsabc) [reply destinations/advertising/fb-custom-audience.mdx:108]: @ishachopra0401 Can we make this minor modification

Comment (ishachopra0401) [reply destinations/advertising/fb-custom-audience.mdx]: updated

**Review by utsabc — APPROVED** [review]
LGTM

**Review by ameypv-rudder — APPROVED** [review]

**Review by ItsSudip — APPROVED** [review]

**Pull request merged**